src/lib.rs

```diff
@@ -307,13 +316,13 @@ pub trait ColorSpace {
 ///assert_eq!(a.mix(&b, 0.5), Rgb::linear_rgb(0.5, 0.5, 0.5));
 ///assert_eq!(a.mix(&b, 1.0), b);
 ///```
-pub trait Mix {
+pub trait Mix<T:Float> {
```

Comment:
That looks just right. I wonder why it would complain... Alright, we can leave it as it is and deal with that later.